libs/sr/page-team/src/TeamSelectors.tsx

```
@@ -0,0 +1,78 @@
+  teamMetadataIndex: number
+  teamId: string
+}) {
+  const { database } = useDatabaseContext()
+  const team = useTeam(teamId)!
+
+  function setCharKey(cKey: CharacterKey | '') {
+    // Remove teammate
+    if (cKey === '') {
+      database.teams.set(
+        teamId,
+        (team) => (team.teamMetadata[teamMetadataIndex] = undefined)
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)